src/readme.md

```diff
@@ -2,21 +2,23 @@
 
 These folders contain the core components of BaGet:
 
-* `BaGet` - the app's entry point that glues everything together
-* `BaGet.Core` - BaGet's core logic and services
-* `BaGet.Core.Server` - the services that implement [the NuGet server APIs](https://docs.microsoft.com/en-us/nuget/api/overview) using `BaGet.Core`
-* `BaGet.Protocol` - libraries to interact with [NuGet servers' APIs](https://docs.microsoft.com/en-us/nuget/api/overview)
-* `BaGet.UI` - BaGet's React frontend
+* `BaGet` - The app's entry point that glues everything together.
+* `BaGet.Core` - BaGet's core logic and services.
+* `BaGet.Hosting` - The services that implements [the NuGet server APIs](https://docs.microsoft.com/en-us/nuget/api/overview) using `BaGet.Core`.
```

Comment:
```suggestion
* `BaGet.Hosting` - The services that implement [the NuGet server APIs](https://docs.microsoft.com/en-us/nuget/api/overview) using `BaGet.Core`.
```

This should be `implement` since `services` is plural

Reply:
I read `the service` instead of `the services`... 😄